### PR TITLE
Curate more publications

### PR DESCRIPTION
**Pull request by cthoyt**
This PR does the following:

- [x] Adds the first publication for BCO @ramonawalls
- [x] Makes publications structured in EMAPA @tfhayamizu. Is there a preferred publication we can mark?
- [x] Updates old reference to proper pubmed one in ICEO @Lemon-Liu

All three maintainers have given their feedback that has been incorporated in this PR.

### CODE REVIEW

File: ontology/iceo.md [hunk 1]
```diff
@@ -16,21 +16,14 @@ license:
...
 repository: https://github.com/ontoice/ICEO
 tracker: https://github.com/ontoice/ICEO/issues
 activity_status: active
 ---
 
-# ICEO: Integrative and Conjugative Element Ontology
-
 ICEO is a community-driven ontology to represent, standardize, and integrate bacterial integrative and conjugative elements (ICEs) and to support computer-assisted reasoning.
 
 The integrative and conjugative elements (ICEs) are modular mobile genetic elements that are integrated into a host genome and are passively propagated during chromosomal replication and cell division. We generated an ICE database called ICEberg (http://db-mml.sjtu.edu.cn/ICEberg/). The ICEO is developed to first systematically represent the ICE information collected in the database. The ICEO will also be further used to enhance the ICEberg and support various applications.
```
Comment (cthoyt) [inline ontology/iceo.md:29]: @Lemon-Liu interestingly, I don't see ICEO used at all in ICEberg. Is this an oversight?

**Comment by tfhayamizu** [comment]
Sorry for the delayed response. The relevant publications are:

    [1] Bard JBL, Kaufman MH, Dubreuil C, Brune RM, Burger A, Baldock RA, Davidson DR (1998). An internet-accessible database of mouse developmental anatomy based on a systematic nomenclature. Mech Dev 74, 111-120.

https://www.sciencedirect.com/science/article/pii/S0925477398000690?via%3Dihub

    [2] Hayamizu TF, Wicks MN, Davidson DR, Burger A, Ringwald M, Baldock RA (2013). EMAP/EMAPA ontology of mouse developmental anatomy: 2013 update. Journal of Biomedical Semantics 4:15.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3851555/

    [3] Hayamizu TF, Baldock RA, Ringwald M (2015). Mouse Anatomy Ontologies: Enhancements and Tools for Exploring and Integrating Biomedical Data. Mamm Genome 26, 422-30.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4602063/

    [4] Hayamizu TF, Ringwald (2016). Textual Anatomics: the Mouse Developmental Anatomy Ontology and the Gene Expression Database for Mouse Development (GXD). In: Kaufman’s Atlas of Mouse Development Supplement. Academic Press.

https://www.sciencedirect.com/science/article/pii/B9780128000434000233

These are listed in chronological order. #4 is the most recent, but may not be widely accessible.

File: ontology/iceo.md (hunk 1 above)
Comment (Lemon-Liu) [reply ontology/iceo.md:29]: Hi @cthoyt, I am sorry for the late response and I appreciate your question. ICEO was developed to ontologically represent and integrate the ICE gene information and functional modules to support automatically computer-assisted reasoning. And ICEO and ICEO-based features are being explored to be integrated into ICEberg. For example,
- We recently developed an [ICEberg integrated SPARQL query interface](https://bioinfo-mml.sjtu.edu.cn/ICEberg2/search_sparql.php) with some useful query models. Users can implement some commonly used SPARQL queries toward ICEO by selecting or inputting keywords without the need of learning and writing SPARQL query scripts. And more query models will be added under the users’ requests. 
- We linked each ICEberg item and the ICEO contents to make a seamless interaction between these two resources. Using [ICEKp1 as an example](https://bioinfo-mml.sjtu.edu.cn/ICEberg2/feature_page.php?ice_id=126), a click of the ICEO ID on the ICEberg page allows a user to access the Ontobee website and identify more detailed information about the ICE. 

And the relevant work has been published recently: 
 - Meng Liu#, Jialin Liu#, Guitian Liu, Hui Wang, Xiaoli Wang, Zixin Deng, Yongqun He, Hong-Yu Ou. ICEO, a biological ontology for representing and analyzing the bacterial integrative and conjugative element. Scientific Data, 2022, 9(1):11.

Comment (Lemon-Liu) [reply ontology/iceo.md:29]: And the relevant link is https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8776819/. I just found out that I may miss the proper information about adding the publication, could you please help add this one for ICEO?

Comment (cthoyt) [reply ontology/iceo.md:29]: That's great, thanks. This corresponds to the publication I already found and added in this PR!

Comment (cthoyt) [reply ontology/iceo.md:29]: Glad to see ICEO has been recently integrated in ICEberg, too :) what a cool resource and such interesting biology I got to learn about from your work

**Review by matentzn — APPROVED** [review]

**Comment by matentzn** [comment]
I assume @cthoyt from your comment in the first comment box that you got permission from all 3 maintainers. Hope that is correct!

**Pull request merged**